credentials/apps/api/v2/filters.py

```
@@ -8,8 +8,14 @@
 
 class ProgramRelatedFilter(django_filters.Filter):
     def filter(self, qs, value):
-        program = Program.objects.filter(uuid=value).first()
-        course_runs = [run.key for run in program.course_runs.all()] if program else []
+        try:
+            program = Program.objects.filter(uuid=value).first()
+        except:  # nopep8 # pylint: disable=bare-except
```

Using a bare except is not good in python, can we simplify this by catching only the raised exception here on invalid UUID?

---

changed the bare `except` with `except ValidationError`.